sedaro/src/sedaro/block_client.py

```diff
@@ -126,6 +164,8 @@ def update(self, **fields) -> 'BlockClient':
         """
         if is_empty(fields):
             raise ValueError(f'Must provide fields to update on the {self.type}.')
+        if ID in fields:
```

Comment:
I don't think I'm a fan of this.  Users should be able to do `block.update(**block)` in my opinion.  Could we instead validate that the ID in `fields` is the same as the ID of the block referenced by `self`?

Reply:
Done!